bodhi/client/__init__.py

```diff
@@ -84,6 +103,23 @@ def new(user, password, **kwargs):
 
 
 @updates.command()
+@add_options(new_edit_options)
+@click.argument('builds')
```

Comment:
I apologize again for not thinking of this sooner, but I think it should be pretty doable.

BTW, I have an open PR that makes it so the CLI can be told to connect to the local development server - it might be helpful for you as you work on this: #1314. With that PR the Vagrant environment automatically uses the local server instead of production, or you can set the BODHI_URL environment variable or the ```--url``` flag to whatever you like.

You can also achieve this by editing the ```__init__()``` method in ```bodhi.client.bindings``` without the PR, but you have to be careful not to commit that change ☺